api/util.py

```diff
@@ -174,6 +174,18 @@ def sanitize_string_to_filename(value):
     keepcharacters = (' ', '.', '_', '-')
     return "".join([c for c in value if c.isalnum() or c in keepcharacters]).rstrip()
 
+def humanize_validation_error(val_err):
+    """
+    Takes a jsonschema.ValidationError, returns a human-friendly string
+    """
+
+    key = 'none'
+    if len(val_err.relative_path) > 0:
+        key = val_err.relative_path[0]
+    message = val_err.message.replace("u'", "'")
+
+    return 'Gear manifest does not match schema on key ' + key + ': ' + message
```

Comment:
This should be useful in more than just gear validation errors. If its going to live in general util can we adjust it a bit to not mention gear manifests or take an optional string param that can be inserted into the message response?

Reply:
Mmm, fair, I'll drop the gear specific part